### DISCUSSION TITLE
Bromite update date is incorrectly showing in F-Droid

### DISCUSSION THREAD

**Discussion by oldphones68** [bromite/bromite#1759] [General]
Bromite update date is incorrectly showing in F-Droid.
It is showing as _1/20/70_ that is incorrect.
What can be the reason behind it?
Is there any other people seeing this happening? 

SS attached for further details. 
![IMG_20220206_124837](https://user-images.githubusercontent.com/51379704/152670668-46e06bef-19d3-4073-97bb-7050d0c9a158.png)

**Comment by ghost** [comment]
![Screenshot_2022-02-06-16-19-18-55](https://user-image.githubusercontent.com/50813528/152674620-b229423e-7fe0-4742-9f71-fa7f7cf3892e.jpg)

@oldphones68  i can confirm it too 
and even confusing why I can't install the update no matter what? @csagan5

**Comment by ghost** [comment]
This is an F-Droid bug,  i guess, for updating Bromite you can use [Github Releases](https://github.com/bromite/bromite/releases), and Bromite also supports In-App Update so you can also use it instead of Github Releases Or F-Droid.

**Reply by oldphones68** [reply]
> Bromite also supports In-App Update

Is that stable? @ASTechnical

**Reply by ghost** [reply]
O yeah. I am also facing such issue. I have installed it from github release. But seeing the problem of Date still exists, I have tried to create a new issue in Github Bromite repo but it was closed by the BOT as I did not maintain the template properly. I have started a new discussion although, I actually did not notice your Discussion.

**Reply by ghost** [reply]
> Is that stable?

Yes.

**Comment by ghost** [comment]
I have also created a new issue in Gitlab F-droid repo in details. Let's see when it gets fixed....

**Reply by MJC579** [reply]
I couldn't update it using the in-app update feature or F-Droid. I opened Uptodown & it got the job done.

**Reply by oldphones68** [reply]
@MJC579 what is *Uptodown*?

**Reply by oldphones68** [reply]
@Bantul-the-great
> I have also created a new issue in Gitlab F-droid repo in details.

Could you please share the link of that issue in GitLab?

**Reply by DerSaxxe** [reply]
https://github.com/bromite/bromite/issues/1768

The issue was reopened by @csagan5. I hope he will find the reason soon.

**Reply by MJC579** [reply]
Sorry, I just noticed you asked about Uptodown appstore. But it's one of the coolest alternative app stores for Android & also Windows. I would strongly recommend anyone check it out & grab their app. If there is an alternate, De-Googled version of an app that will be the version Uptodown has. Lots of good stuff in there. I use it regularly & I've never contracted a bug from them. Not only is it one of the better alternate app stores for Android but it is one of the very few alternate app stores for Windows. I'm quite surprised that you aren't already aware of the Uptodown appstore.

**Comment by oldphones68** [comment]
Now I'm seeing it is as the SS.
![IMG_20220214_111341](https://user-images.githubusercontent.com/51379704/153804151-5e466683-9c4a-472c-89ac-e09d5fe36432.jpg)

**Reply by csagan5** [reply]
I cannot reproduce this; try adding the repository from scratch as explained here: https://www.bromite.org/fdroid

**Reply by MJC579** [reply]
All you have to do is copy the URL, go into F-Droid's settings, tap repositores, tap the + button in the upper right hand corner & your copied URL should automatically be filled in already. If it isn't just long press on that line & paste it. Once it's there though just hit the add button or whatever & your done. Alternatively, you could just use Aurora Droid. An alternative, expanded version of F- Droid with all the popular repos already saved in it. You just have to flip the enable all repos switch on in the settings & save your settings.  Download AuroraDroid stable. I don't even have the original F-Droid app anymore. Just use this one now. 
https://auroraoss.com/
